### DISCUSSION TITLE
Add the possibility to fix dates in immich based on the google takeout info

### DISCUSSION THREAD

**Discussion by simulot** [simulot/immich-go#89] [Ideas]
Have you already uploaded a ton of photos in immich without using immich-go, and you wish to correct their date based on a information collected in google photos?

**Comment by caddish** [comment]
Yes I would like that very much, I have manually merged my old local backup with a previous google photo takeout but I'm missing lots of dates especially on videos. And would like to use a current google photos takeout to update missing dates and album

**Reply by simulot** [reply]
Immich has added a couple of version ago the possibility to update date and GPS coordinates through the API. I'm thinking about a command to refresh immich metadata using a google photo archive.

**Comment by simulot** [comment]
The coming release  will implement this

**Reply by dreautall** [reply]
Hi @simulot,

if I re-run immich-go on the takeout, will it just update all the metadata (specifically the date) on existing files? Is there an option for it? Or is it not yet supported? I unfortunately did the first import before the date was taken from google Takeout, and deleting the complete archive seems quite unpractical.

Thank you for the great software!
